clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/DefaultIngestionBackend.java

```diff
@@ -86,7 +86,7 @@ public void startConsumption(VeniceStoreVersionConfig storeConfig, int partition
           storeAndVersion.getFirst(),
           storeAndVersion.getSecond().getNumber(),
           partition,
-          serverConfig.getBlobTransferDisabledOffsetLagThreshold());
+          storeConfig.getBlobTransferDisabledOffsetLagThreshold());
```

Comment:
Any reason for this change?

Reply:
Also can we rename serverConfig method name to `getServerLevelBlobTransferDisabledOffsetLagThreshold` or something?

Reply:
Synced offline with Jingyan, we don't need this change. We need to add to global-config for DVC to pick up override.

Reply:
This issue is related to the deployment not successfully picking up the getBlobTransferDisabledOffsetLagThreshold config (unrelated to the fix for the integration test). The root cause is that the global-config was not added, not related to the code issue. I will revert this change.